features/admin.applications.v1/components/forms/inbound-saml-form.tsx

```diff
@@ -1734,82 +1789,91 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
                                 </Grid.Column>
                             </Grid.Row>
                             <div ref={ returnToURL }></div>
-                            <URLInput
-                                urlState={ returnToURLS }
-                                setURLState={ (url: string) => {
-                                    setReturnToURLS(url);
-
-                                    if (initialValues
-                                        ?.singleLogoutProfile
-                                        ?.idpInitiatedSingleLogout?.returnToUrls?.toString() !== url) {
-                                        setIsFormStale(true);
-                                    }
-                                } }
-                                labelName={
-                                    t("applications:forms.inboundSAML.sections" +
-                                        ".idpInitiatedSLO.fields.returnToURLs.label")
-                                }
-                                value={
-                                    initialValues?.singleLogoutProfile.idpInitiatedSingleLogout.returnToUrls.toString()
-                                }
-                                placeholder={
-                                    t("applications:forms.inboundSAML.sections" +
-                                        ".idpInitiatedSLO.fields.returnToURLs.placeholder")
-                                }
-                                validationErrorMsg={
-                                    t("applications:forms.inboundSAML.sections" +
-                                        ".idpInitiatedSLO.fields.returnToURLs.validations.invalid")
-                                }
-                                validation={ (value: string) => {
-
-                                    let label: ReactElement = null;
-
-                                    if (!URLUtils.isHttpUrl(value) && !URLUtils.isHttpsUrl(value)) {
-                                        label = (
-                                            <Label basic color="orange" className="mt-2">
-                                                { t("console:common.validations.unrecognizedURL.description") }
-                                            </Label>
-                                        );
-                                    }
+                            <Grid.Row
+                                columns={ 1 }
+                                data-componentid="application-edit-inbound-saml-form-slo-return-to-urls"
```

Comment:
shall we use the already defined `testId` to construct the new component ids?

```suggestion
                                data-componentid={ testId + "-slo-return-to-urls" }
```

Reply:
It is best to avoid using dynamic component IDs for these fields, as they are referenced in templates, and any changes to these components will impact all SSO templates.